ibm_catalog.json

```diff
@@ -65,34 +66,43 @@
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "service_name": "all-account-management-services"
+              "service_name": "all-account-management-services",
+              "notes": "Required for Account Management Services."
```

Comment:
Why would we need such wide access just to provision and configure redis?? The only account management permissions I would be aware of would be Viewer access on the resource group you want to create the instance in. And to document such a permissions you would do this:
```
{
      "role_crns": [
        "crn:v1:bluemix:public:iam::::role:Viewer"
      ],
      "service_name": "Resource group only",
      "notes": "Viewer access is required in the resource group you want to provision in."
    }
```
(as per point 4 in https://github.ibm.com/GoldenEye/issues/issues/13599).